src/Backend/Modules/MediaGalleries/Domain/MediaGallery/MediaGallery.php

```diff
@@ -0,0 +1,403 @@
+<?php
+
+namespace Backend\Modules\MediaGalleries\Domain\MediaGallery;
+
+use Ramsey\Uuid\Uuid;
+use Doctrine\ORM\Mapping as ORM;
+use Backend\Core\Engine\Model;
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroup;
+use Common\ModuleExtraType;
+
+/**
+ * MediaGallery
+ *
+ * @ORM\Entity(repositoryClass="Backend\Modules\MediaGalleries\Domain\MediaGallery\MediaGalleryRepository")
+ * @ORM\HasLifecycleCallbacks
+ */
+class MediaGallery
+{
+    /**
+     * @var string
+     *
+     * @ORM\Id()
+     * @ORM\GeneratedValue(strategy="UUID")
+     * @ORM\Column(type="guid")
+     */
+    private $id;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(type="integer")
+     */
+    protected $userId;
+
+    /**
+     * @var integer
+     *
+     * ToDo: when Fork CMS has ModuleExtra entity in core, use its Entity as type
+     *
+     * @ORM\Column(type="integer")
+     */
+    protected $moduleExtraId;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    protected $action;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    protected $title;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", nullable=true)
+     */
+    protected $text;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(type="datetime")
+     */
+    protected $createdOn;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(type="datetime")
+     */
+    protected $editedOn;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(type="datetime")
+     */
+    protected $publishOn;
+
+    /**
+     * @var Status
+     *
+     * @ORM\Column(type="media_gallery_status")
+     */
+    protected $status;
+
+    /**
+     * @var MediaGroup
+     *
+     * @ORM\OneToOne(
+     *      targetEntity="Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroup",
+     *      cascade="persist",
+     *      orphanRemoval=true
+     * )
+     * @ORM\JoinColumn(
+     *      name="mediaGroupId",
+     *      referencedColumnName="id",
+     *      onDelete="cascade"
+     * )
+     */
+    protected $mediaGroup;
+
+    /**
+     * MediaGallery constructor.
+     *
+     * @param string $title
+     * @param string $action
+     * @param int $userId
+     * @param \DateTime $publishOn
```

Comment:
For example: you can create a photo slider, but only want to let it be visible next week.
Ok?